packages/nodes-base/nodes/Facebook/FacebookTrigger.node.ts

```diff
@@ -195,6 +195,7 @@ export class FacebookTrigger implements INodeType {
 					throw new NodeOperationError(
```

Comment:
I'm wondering if we should create a `NodeOperationWarning` (and maybe `NodeApiWarning`), as people could forget to add `severity`, even when they know the severity.

Reply:
I was also thinking of a more general solution because this is neverending :)

Maybe we could have a separate `nodes` project in Sentry, and flip it so we only report errors there intentionally, e.g. `ReportableNodeOperationError` and `ReportableNodeApiError` - this is assuming the majority of errors in `nodes-base` are likely to be non-reportable because they come from external APIs, but I could be wrong to assume this. 

Something to discuss with nodes team.